collect_stata/read_stata.py

```diff
@@ -38,16 +39,11 @@ def parse_file(self):
         self.metadata = self.generate_tdp()
 
     def generate_tdp(self):
-        """
-        Generate tabular data package file
-
-        Input:
-        datatable: pandas DataFrame
-        stata_name: string
-        data: pandas StataReader
+        """Generate tabular data package file.
 
-        Output:
-        tdp: dict
+        Returns:
+            datatable (pandas.DataFrame): Extracted data.
+            metadata (dict): Meta information for the data.
```

Comment:
This does not generate a file.
It also does not return a pandas.DataFrame and a dictionary.

Reply:
Use a type hint for the function, so you don't have to specify it again in the docstring.
Only write normal text for `Returns:`. This text should explain what is returned,